apm-sniffer/apm-toolkit-activation/apm-toolkit-trace-activation/src/main/resources/skywalking-plugin.def

```diff
@@ -19,4 +19,10 @@ toolkit-trace=org.apache.skywalking.apm.toolkit.activation.trace.TraceAnnotation
 toolkit-trace=org.apache.skywalking.apm.toolkit.activation.trace.TraceContextActivation
 toolkit-trace=org.apache.skywalking.apm.toolkit.activation.trace.CallableOrRunnableActivation
 toolkit-tag=org.apache.skywalking.apm.toolkit.activation.trace.TagAnnotationActivation
-toolkit-exception=org.apache.skywalking.apm.toolkit.activation.trace.IgnoredExceptionAnnotationActivation
\ No newline at end of file
+toolkit-exception=org.apache.skywalking.apm.toolkit.activation.trace.IgnoredExceptionAnnotationActivation
+toolkit-tracer=org.apache.skywalking.apm.toolkit.activation.trace.CarrierItemRefActivation
```

Comment:
This should be added into plugin-list.md, as a new name means these instrumentations could be disabled through this name.
So we check consistent with docs to keep updated.

Reply:
Ok, I will add `toolkit-tracer` to the plugin-list.md.